docs/paths/requests.yml

```diff
@@ -0,0 +1,98 @@
+PATH_requests:
+  post:
+    summary: Create new request
+    description: Used to create a new request in collection.
+    requestBody:
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/_index.yml#/createRequestPayload"
+    responses:
+      '201':
+        description: 'Created'
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                message: 
+                  type: string
+            example:
+              type: object
+              properties:
+                message: "Request has successfully been created."
+
+  get:
+    summary: Find a request
```

Comment:
instead of GET on a single request, we should have a function like `GET /request/{spaceId}` for all the requests for a given space